roles/openshift_node/tasks/main.yml

```diff
@@ -72,7 +79,7 @@
 - name: Reload systemd units
   command: systemctl daemon-reload
   when: openshift.common.is_containerized and ( ( install_node_result  | changed )
-    or ( install_ovs_result | changed ) )
+    or ( install_ovs_result | changed ) or ( install_ovs_sysconfig | changed ) )
```

Comment:
daemon-reload shouldn't need to be invoked when the sysconfig file is changed, but you will want to trigger a handler  to restart the openvswitch service